resolwe/flow/tests/__init__.py

```diff
@@ -59,23 +59,6 @@ def _register_schemas():
         SCHEMAS_FIXTURE_CACHE['descriptor_schemas'] = list(DescriptorSchema.objects.all())
 
 
-flow_executor_settings = settings.FLOW_EXECUTOR.copy()
-# since we don't know what uid/gid will be used inside Docker executor, others
-# must have all permissions on the data directory
-flow_executor_settings['DATA_DIR_MODE'] = 0o777
-# create a temporary upload directory so we can give it relaxed permissions and
-# make it convenient for use by Docker
-flow_executor_settings['UPLOAD_PATH'] = os.path.join(flow_executor_settings['UPLOAD_PATH'], 'test_upload')
```

Comment:
I like that test upload files go to a separate directory. I would do the sample thing for test data folders. This is just my opinion, you can decide how you want.